lntest/itest/lnd_test.go

```diff
@@ -14207,32 +14207,50 @@ func TestLightningNetworkDaemon(t *testing.T) {
 	aliceBobArgs := []string{
 		"--default-remote-max-htlcs=483",
 	}
-	if err = lndHarness.SetUp(aliceBobArgs); err != nil {
-		ht.Fatalf("unable to set up test lightning network: %v", err)
-	}
 
 	t.Logf("Running %v integration tests", len(testsCases))
 	for _, testCase := range testsCases {
-		logLine := fmt.Sprintf("STARTING ============ %v ============\n",
-			testCase.name)
+		testCase := testCase
 
-		err := lndHarness.EnsureConnected(
-			context.Background(), lndHarness.Alice, lndHarness.Bob,
-		)
-		require.NoError(t, err, "unable to connect alice to bob")
+		success := t.Run(testCase.name, func(tt *testing.T) {
+			cleanTestCaseName := strings.ReplaceAll(
+				testCase.name, " ", "_",
+			)
+
+			err = lndHarness.SetUp(cleanTestCaseName, aliceBobArgs)
```

Comment:
would an alternative be to set up completely new nodes before each test case? (perhaps a bigger change)

Reply:
I may be misunderstanding your comment, but I think that's what we do? `SetUp` starts the two nodes (Alice and Bob), while `TearDown` just stops them and deletes all their files. This cycle is done before/after each testcase.

Reply:
Ah, I thought they would just be restarted 👍